tracer_test.go

```diff
@@ -574,11 +574,25 @@ func TestTracerDefaultTransport(t *testing.T) {
 }
 
 func TestTracerUnsampledTransactions(t *testing.T) {
+	newTracer := func(v int, err error) (*apm.Tracer, *transporttest.ServerVersionRecorderTransport) {
+		transport := transporttest.ServerVersionRecorderTransport{
```

Comment:
does ServerVersionRecorderTransport need to be in transporttest, or could we move it to to tracer_test.go and unexport it?